habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -474,7 +474,7 @@ class HumanoidJointAction(RobotAction):
     def __init__(self, *args, sim: RearrangeSim, **kwargs):
         super().__init__(*args, sim=sim, **kwargs)
         self._sim: RearrangeSim = sim
-        self.num_joints = 19
+        self.num_joints = 17
```

Comment:
Should this be configurable from the action config?

Reply:
True, modified